src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -172,4 +197,46 @@ public function get_fields(): array {
 		 */
 		return apply_filters( 'tec_tickets_emails_settings_fields', $fields );
 	}
+
+	/**
+	 * Check if currently editing email.
+	 * 
+	 * @since TBD
+	 *
+	 * @param Email_Abstract $email
+	 * 
+	 * @return boolean
+	 */
+	public function is_editing_email( $email = null ) {
+		if ( ! $this->is_on_tab() ) {
+			return false;
```

Comment:
By using the section [from here](https://github.com/the-events-calendar/event-tickets/pull/2550/files#diff-b864484972892032b7a96254575015028df2c5156fd149628898c69c2b99e626R129) we shouldn't have to check if is on tab.